### PR TITLE
Clarified supported Ecma edition for regex

### PR DESCRIPTION
**Pull request by dvh**
Added supported Ecma edition (5.1) for regular expressions in the link text and used the formal name: [Ecma-262 Edition 5.1](https://www.ecma-international.org/ecma-262/5.1/).

See also: https://github.com/OAI/OpenAPI-Specification/issues/1687

### CODE REVIEW

File: versions/3.0.3.md [hunk 1]
```diff
@@ -2284,7 +2284,7 @@ The following properties are taken directly from the JSON Schema definition and
 - exclusiveMinimum
 - maxLength
 - minLength
-- pattern (This string SHOULD be a valid regular expression, according to the [ECMA 262 regular expression](https://www.ecma-international.org/ecma-262/5.1/#sec-7.8.5) dialect)
+- pattern (This string SHOULD be a valid regular expression, according to the [Ecma-262 Edition 5.1 regular expression](https://www.ecma-international.org/ecma-262/5.1/#sec-7.8.5) dialect)
```
Comment (earth2marsh) [inline versions/3.0.3.md]: `Ecma-` should be `ECMA-`, no?

Comment (dvh) [reply versions/3.0.3.md]: Hm, not sure... https://www.ecma-international.org/ecma-262/5.1/#sec-7.8.5 says:

> This is the HTML rendering of *Ecma-262 Edition 5.1,* The ECMAScript Language Specification.

It looks like ECMA is uppercase mostly when used in 'ECMAScript'.

File: versions/3.0.3.md [hunk 2]
```diff
@@ -2299,7 +2299,7 @@ The following properties are taken directly from the JSON Schema definition and
 - exclusiveMinimum
 - maxLength
 - minLength
-- pattern (This string SHOULD be a valid regular expression, according to the [ECMA 262 regular expression](https://www.ecma-international.org/ecma-262/5.1/#sec-15.10.1) dialect)
+- pattern (This string SHOULD be a valid regular expression, according to the [Ecma-262 Edition 5.1 regular expression](https://www.ecma-international.org/ecma-262/5.1/#sec-15.10.1) dialect)
```
Comment (MikeRalphson) [inline versions/3.0.3.md]: Conflict with link fragment change resolved here. Now LGTM.

File: versions/3.0.3.md (hunk 1 above)
Comment (earth2marsh) [reply versions/3.0.3.md]: Wow, what a mess of mixed caps! :)

You're right though, based on their usage in that doc, we can basically do what we want. My own personal sensibilities would argue for capitalization, and the header on that page does use capitalization: 
image

... but then does something else shortly thereafter. My point is definitely a nit, and this shouldn't hold up the acceptance, though if you don't mind using `ECMA` instead, please do, thx.

File: versions/3.0.3.md (hunk 2 above)
Comment (earth2marsh) [reply versions/3.0.3.md]: Thanks Mike. I discussed with @webron , too, and ready to merge.

**Pull request merged**